app/views/qsets/_qset_context.html.erb

```diff
@@ -1,32 +1,28 @@
 <span id="qset_context">
-  <h4>
-    Qset: <%= @qset.name %>
-    <% if can? :update, Qset %>
-      <a class="btn shortcut-button-link"
-         id="edit-qset"
-         data-toggle="modal"
-         data-target="#modal-edit-qset">
-        <span class="glyphicon glyphicon-edit shortcut-button-glyphicon"></span>
-      </a>
-      <!-- todo: delete button here? -->
+    <h3 class="center"><b><%= @qset.name %></b>
+    <% if !@qset.parent.nil? %>
+      <% if can? :update, Qset %>
+        <a class="btn shortcut-button-link"
+           id="edit-qset"
+           data-toggle="modal"
+           data-target="#modal-edit-qset">
+          <span class="glyphicon glyphicon-edit shortcut-button-glyphicon"></span>
+        </a>
+         <!-- todo: delete button here? -->
+      <% end %>
     <% end %>
-  </h4>
-  <% for qset in @qset.ancestors.reverse %>
-    <%= link_to h(qset.name), qset %> &gt;
-  <% end %>
-  <%= @qset.name %>
-  <% unless @qset.children.empty? %>
-    &gt;
-    <%= select(@qset, :qset_id, @qset.children.map {|qs| [qs.name, qs.id, {:'data-qset-url' => qset_path(qs.id)}]}, {prompt: "sub-qsets"}, :id => 'qsets') %>
-  <% end %>
+    <br>
+    <br>
+    </h3>
   <% if can? :create, Qset %>
-    &gt;
+    <a class="center"><b>Classes</b>
```

Comment:
I don't think this is meant to be an `<a>` (anchor tag/link); also this heading will only appear for admins, because it's in the `<% if can? :create, Qset %>` block, no?